code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -758,3 +758,39 @@
 	desc = "A dangerous fusion of flavors!"
 	icon_state = "plasma"
 	list_reagents = list(/datum/reagent/medicine/molten_bubbles/plasma = 50)
+
+/obj/item/reagent_containers/food/drinks/ration
+	name = "Rationed Air"
```

Comment:
why not like. "Generic Ration Drink". Rationed air seems to re-occur as a name later.

Reply:
I forgor to change it but it is not meant to show up in game anyway
Will change